src/components/projects/EditorContainer.tsx

```diff
@@ -30,28 +30,26 @@ export const EditorContainerInner = styled.div`
   display: flex;
   flex-direction: column;
   overflow: hidden;
-  width: 960px;
+  width: 100%;
```

Comment:
adding fixed width will be a problem with multiple screen dimensions, why remove the percentage!
 or we can use the added unit `editorMaxWidth` as `max-width` so it will stay flexible with screen size

Reply:
there is no fix width, it is set to 100% ;)